sections/main-product.liquid

```diff
@@ -484,6 +497,10 @@
           if product.selected_or_first_available_variant.featured_media != null
             assign media = product.selected_or_first_available_variant.featured_media
             render 'product-media', media: media, loop: section.settings.enable_video_looping, variant_image: section.settings.hide_variants
+          else
```

Comment:
Previously you used the check for null `host_product.featured_media != null` , maybe it's wise to do this here too before using the value?

Reply:
Done!